pajbot/models/command.py

```diff
@@ -440,20 +440,23 @@ def autogenerate_examples(self):
             example = CommandExample(self.id, "Default usage")
             subtype = self.action.subtype if self.action.subtype != "reply" else "say"
             example.add_chat_message("say", self.main_alias, "user")
+            cleanstring = re.sub(
+                r"\$\(urlfetch ([A-Za-z0-9\-._~:/?#\[\]@!$%&\'()*+,;=]+)\)", "(urlfetch)", self.action.response
```

Comment:
Would it be possible to import this instead of duplicating it?

Reply:
Yes this shouldn't be a problem - I'm assuming you mean import it from the linkchecker module?
I can work on it tonight

Reply:
I've gotten as far as this, but my bot produces an error 500 on the commands page. Are you able to point me in the right direction?
![image](https://user-images.githubusercontent.com/12804673/72525757-9d23f380-38b0-11ea-8b28-0e75253c90f2.png)